src/entitySearchResultsList.js

```diff
@@ -60,6 +60,9 @@ export default class EntitySearchResultsList extends LightningElement {
                     if(originalValue instanceof Object){
                         key = key.replaceAll("\.", "-");
                     }
+                    
+                    val = val.replace("<br>", " ");
+                    val = val.replace(/<\/?[^>]+(>|$)/g, "");
```

Comment:
Removing html tags only for demo

Reply:
In real life I suppose this is why knowledge behaves differently than other entities